### ISSUE TITLE
Regaring jscrollpane issue

### ISSUE THREAD

**Issue by zico2013** [vitch/jScrollPane#223]
I am using jscrollpane in thickbox but it is not working properly. I am sharing my code let me know if there is any mistake.
<html>
<head>
<link type="text/css" href="style/jquery.jscrollpane.css" rel="stylesheet" media="all" />

<style>
#box5{
            margin:20px 10px 0 20px;
            padding-left:30px;
            float:right;
            width:60%;
            height:70%;
            overflow-y:auto;
            overflow-x:hidden;
            scrollbar-base-color:#FFFFFF;
            text-align:left;
            border-color:#ffdfb2;
            border-left-width:1px;
            border-left-style:solid;            
          
    }
</style>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>

<script type="text/javascript" src="ext/js/jquery-1.9.1.js"></script>

<script type="text/javascript" src="script/jquery.mousewheel.js"></script>

<script type="text/javascript" src="script/jquery.jscrollpane.min.js"></script>

</head>
<body>

<div id="box5"><div class="scroll-pane" style="width:100%; height:100%; overflow:auto;">

<p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in ligula id sem tristique ultrices
                    eget id neque. Duis enim turpis, tempus at accumsan vitae, lobortis id sapien. Pellentesque nec orci
                    mi, in pharetra ligula. Nulla facilisi. Nulla facilisi. Mauris convallis venenatis massa, quis
                    consectetur felis ornare quis. Sed aliquet nunc ac ante molestie ultricies. Nam pulvinar ultricies
                    bibendum. Vivamus diam leo, faucibus et vehicula eu, molestie sit amet dui. Proin nec orci et elit
                    semper ultrices. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus
                    mus. Sed quis urna mi, ac dignissim mauris. Quisque mollis ornare mauris, sed laoreet diam malesuada
                    quis. Proin vel elementum ante. Donec hendrerit arcu ac odio tincidunt posuere. Vestibulum nec risus
                    eu lacus semper viverra.
</p>
<p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec in ligula id sem tristique ultrices
                    eget id neque. Duis enim turpis, tempus at accumsan vitae, lobortis id sapien. Pellentesque nec orci
                    mi, in pharetra ligula. Nulla facilisi. Nulla facilisi. Mauris convallis venenatis massa, quis
                    consectetur felis ornare quis. Sed aliquet nunc ac ante molestie ultricies. Nam pulvinar ultricies
                    bibendum. Vivamus diam leo, faucibus et vehicula eu, molestie sit amet dui. Proin nec orci et elit
                    semper ultrices. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus
                    mus. Sed quis urna mi, ac dignissim mauris. Quisque mollis ornare mauris, sed laoreet diam malesuada
                    quis. Proin vel elementum ante. Donec hendrerit arcu ac odio tincidunt posuere. Vestibulum nec risus
                    eu lacus semper viverra.
</p>
</div></div>

</body>
</html>

If you have any sample code using jscrollpane in thickbox then please attach in reply.

Any help will be appreciated,

Thanks in advance

**Comment by illuusio** [comment]
If this is still issue please write note and have example in some place where I can check it
